moderatorFrontend/build.gradle.kts

```diff
@@ -37,20 +34,16 @@ kotlin {
       dependencies {
         implementation(kotlin("stdlib-js"))
         implementation(project(":common"))
-        implementation(project.dependencies.enforcedPlatform(kotlinw("wrappers-bom:$kotlinWrappersVersion")))
 
         api("org.jetbrains.kotlinx:kotlinx-html-js:$kotlinx_html_version")
 
-        // When updating any of the below kotlinw dependencies, check
-        // https://repo.maven.apache.org/maven2/org/jetbrains/kotlin-wrappers/kotlin-mui-icons/ (or the corresponding url for other packages)
-        // to see which versions are available for which kotlinw version.
```

Comment:
I guess we don't need this comment anymore because for checking the new versions we can use `libs` or?

Reply:
Yes, the version catalog now automatically uses the version specified in `settings.gradle.kts` for all kotlin wrappers dependencies.